src/config/index.js

```diff
@@ -0,0 +1,5 @@
+const config_common = require("./common.json");
+const config_env = require(`./${process.env.NODE_ENV}.json`);
```

Comment:
I'm not totally clear what path data is following around the app here. How does one set `process.env.NODE_ENV`?

Would you be willing to do a code walkthrough of this new front-end template for us at some point? I'd be happy to host you at SCL.

Reply:
https://create-react-app.dev/docs/adding-custom-environment-variables

> There is also a special built-in environment variable called NODE_ENV. You can read it from process.env.NODE_ENV. When you run npm start, it is always equal to 'development', when you run npm test it is always equal to 'test', and when you run npm run build to make a production bundle, it is always equal to 'production'. You cannot override NODE_ENV manually. This prevents developers from accidentally deploying a slow development build to production.

Let me try to build an end-to-end app, and then I am happy to share in SCL. Don't feel having enough knowledge to share now :) But happy to walkthru code with you personally.